.github/workflows/label-mandatory-workspace-prs.yaml

```diff
@@ -29,8 +29,10 @@ jobs:
           script: |
             const fs = require('fs');
             
-            // Read the downstream-plugins file to get required plugins
-            const downstreamPluginsContent = fs.readFileSync('downstream-plugins', 'utf8');
+            // read the 2 rhdh-community-plugins.txt rhdh-downstream-plugins.txt files to get required plugins
+            const downstreamPluginsContent = fs.readFileSync('rhdh-downstream-plugins.txt', 'utf8') + 
+              fs.readFileSync('rhdh-community-plugins.txt', 'utf8');
```

Comment:
Not sure this works here.
Using. the same approach as https://github.com/redhat-developer/rhdh-plugin-export-overlays/pull/1591/files#diff-76169583b7a052a1b4cc61cd96c5dde9a6d2b30848b10a3028a5dc4132b9e3f6R44 wold be better I guess.